airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -374,47 +558,82 @@ definitions:
         type: string
+        examples:
+          - "%Y-%m-%dT%H:%M:%S.%f%z"
       cursor_granularity:
-        description: Smallest increment the datetime_format has (ISO 8601 duration) that is used to ensure the start of a slice does not overlap with the end of the previous one
+        title: Cursor Granularity
+        description:
+          Smallest increment the datetime_format has (ISO 8601 duration) that is used to ensure the start of a slice does not overlap with the end of the previous one, e.g. for %Y-%m-%d the granularity should
+          be P1D, for %Y-%m-%dT%H:%M:%SZ the granularity should be PT1S.
         type: string
+        examples:
+          - "PT1S"
       end_datetime:
-        description: The datetime that determines the last record that should be synced
+        title: End Datetime
+        description: The datetime that determines the last record that should be synced.
         anyOf:
           - type: string
           - "$ref": "#/definitions/MinMaxDatetime"
+        examples:
+          - "2021-01-1T00:00:00Z"
       start_datetime:
-        description: The datetime that determines the earliest record that should be synced
+        title: Start Datetime
+        description: The datetime that determines the earliest record that should be synced.
         anyOf:
           - type: string
           - "$ref": "#/definitions/MinMaxDatetime"
+        examples:
+          - "2020-01-1T00:00:00Z"
       step:
-        description: The size of the time window (ISO8601 duration)
+        title: Step
+        description: The size of the time window (ISO8601 duration).
         type: string
+        examples:
+          - "P1W"
+          - "{{ config['step_increment'] }}"
       end_time_option:
-        description: Request option for end time
+        title: End Time Request Option
+        description: Optionally configures how the end datetime will be sent in requests to the source API.
         "$ref": "#/definitions/RequestOption"
       lookback_window:
-        description: How many days before start_datetime to read data for (ISO8601 duration)
+        title: Lookback Window
+        description: Time interval before the start_datetime to read data for, e.g. P1M for looking back one month.
         type: string
+        examples:
+          - "P1D"
+          - "P{{ config['lookback_days'] }}D"
       partition_field_end:
-        description: Partition start time field
+        title: Partition Field End
+        description: Name of the partition start time field.
         type: string
+        examples:
+          - "ending_time"
       partition_field_start:
-        description: Partition end time field
+        title: Partition Field Start
+        description: Name of the partition end time field.
         type: string
+        examples:
+          - "starting_time"
       start_time_option:
-        description: Request option for start time
+        title: Start Time Request Option
```

Comment:
Is called `Inject start time into outgoing HTTP request`

Reply:
updated